prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Steve Smith';
+    console.log(fullName);
+    var numberOfPets = 0;
+    console.log(numberOfPets);
+    var likesVegetables = true;
+    console.log('likes Vegetables = ' + likesVegetables);
+    var nothing = null;
+    console.log(nothing);
+    var never = null;
```

Comment:
for this `never` variable, you can declare it alone and it will receive a value of `undefined` automatically

```js
var never;
```